src/main/python/analyzers/embedded_analyzer.py

```diff
@@ -0,0 +1,35 @@
+import tensorflow_hub as hub
+import pandas as pd
+
+
+print('Loading global sentence embedder...')
+# module_url = "https://tfhub.dev/google/universal-sentence-encoder/4"
+module_url = "/home/minasonbol/PycharmProjects/Text_Mining/universal-sentence-encoder_4"
```

Comment:
I feel like this won't work on every machine. Is this a url or a file path? If you're looking to use a file path, look at what I did in nba_commisioner line 9.

Reply:
in my initial push, i had this line 7 hashed out, and was using the url in line 6. when i was demonstrating the code for you guys i used the local version, because it is much faster to use the locally stored version of the model than to download it everytime (900+ MB). i'll keep only line 6

Reply:
Ah okay. Well it's fine if you want to use a local version to increase speed, you just have to use something called a relative file path. The relative path gives the file location from the current file you are in. That is desirable for projects like this because many people will run the code from different machines. Line 9 in nba_commisioner shows how I was able to use a relative file path. You will probably need an extra `.parent`